### PR TITLE
Fixed issues with federated foreign productVariant

### PR DESCRIPTION
**Pull request by KiLanger**
This PR fixes the remaining issues regarding federation.
Code duplication regarding the EntityConnection will be adressed in a sperate PR since it is a seperate issue.

https://frontend.gropius.duckdns.org/components/9d12d6ac-e85a-407c-bdff-a8321fb75d3a/issues/53f91620-749e-4258-a38d-e75cc1f78db4
 

- [x] Requirements of the issue are met
- [x]  Code has been reviewed
- [x]  Code is documented
- [x]  GraphQL related artefacts are documented

### CODE REVIEW

File: src/inventory.gql [hunk 1]
```diff
@@ -121,7 +121,7 @@ type Mutation {
 """The input of a product item batch creation"""
 input CreateProductItemBatchInput {
   """The product variant id of the product item"""
-  productVariantId: UUID!
```
Comment (nk-coding) [inline src/inventory.gql:124]: Why rename these inputs?
The schema all other services use includes the Id suffix for input ids

**Review by nk-coding — CHANGES_REQUESTED** [review]

File: src/inventory.gql [hunk 2]
```diff
@@ -54,10 +54,10 @@ type Query {
     orderBy: ProductItemOrder
   ): ProductItemConnection!
 
-  """Returns product items in inventory of a product variant version"""
-  productItemsByProductVariantId(
+  """Returns product items in inventory of a product variant"""
+  productItemsByProductVariant(
```
[the remark this answers is not in the document]
Comment (nk-coding) [reply src/inventory.gql:58]: ok I found the connection there, my bad
what is this for then?
looks a bit duplicated if we already have the connection on ProductVariant

File: src/inventory.gql (hunk 1 above)
Comment (KiLanger) [reply src/inventory.gql:124]: agreed for consistency with other services

File: src/inventory.gql (hunk 2 above)
[the remark this answers is not in the document]
Comment (KiLanger) [reply src/inventory.gql:58]: image
I simply misspelled this at some point, it always has been to product variant

[the remark this answers is not in the document]
Comment (KiLanger) [reply src/inventory.gql:58]: conclusion: productItemsByProductVariantId will be removed in the rework

**Review by nk-coding — APPROVED** [review]

**Pull request merged**